app/models/explainer.rb

```diff
@@ -107,7 +107,7 @@ def self.search_by_similarity(text, language, team_id, limit)
     models_thresholds = Explainer.get_alegre_models_and_thresholds(team_id)
     context = {
       type: 'explainer',
-      team: Team.find(team_id).slug
+      team: team_id
```

Comment:
As above, I'd rename `team` to `team_id` to be consistent with how we index fact-checks

Reply:
Since I was still here... done!